Improve the new user experience

milljm opened the pull request:
A better FAQ, Troubleshooting, Getting Started experience

Ref #12328

Commit:
Improve the new user experience

A better FAQ, Troubleshooting, Getting Started experience

Ref #12328

milljm commented:
I am cool with being asked for additions before this gets merged. Creating a pull request to get things moving...

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/246686/) on 6fc1a10 wanted to post the following:

View the site [here](http://mooseframework.inl.gov/docs/PRs/12355/site)

This comment will be updated on new commits.

Commit:
Modify documentation related links

Add Doxygen link
Move Source and Syntax from documentation drop down to Application/Framework development

Commit:
Add a HPC User 'getting started' page

Closes #12404

Commit:
Add 'what now' sections to HPC Users page

Closes #12404

Commit:
restructure some wordage a bit

Commit:
Additional awesome changes

Remove the Cluster Admin specific instructions from the drop down.
Rename and modify the Cluster Users instructions to be much more attuned to a normal Cluster user.
Modify the troubleshooting page to make use of the !package! include routines.

Commit:
Adjust menu font size

Make header size consistent throughout the Getting Starting instructions

milljm commented:
@friedmud If you would review the Cluster Users page in particular... Let me know if this is what you want.

Commit:
Adjust the font weight css

There are elements which are very difficult to read:

  https://mooseframework.inl.gov/source/kernels/PrimaryTimeDerivative.html
  https://mooseframework.inl.gov/syntax/index.html

This modification will help with that.

Made the links easier to read.

Commit:
Fix grammar/spelling errors

permcody commented:
Testing....
